index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Testing</title>
+</head>
+
+<body>
+    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Vel illo voluptatum ipsa odio quisquam obcaecati
+        necessitatibus delectus dignissimos! Officiis qui rerum eligendi totam voluptate consequuntur optio a laudantium
+        aut quisquam.</p>
+</body>
```

Comment:
This time is fine, I will accept it, but in the future, please indent with 2 spaces only!